partcad-cli/src/partcad_cli/click/commands/supply/find.py

```
@@ -0,0 +1,68 @@
+@click.option(
+    "--provider",
+    help="Provider to use",
+    show_envvar=True,
+)
+@click.argument(
+    "specs",
+    metavar="object[[,material],count]",
+    nargs=-1,
+)  # help="Part (default) or assembly to quote, with options"
+@click.pass_obj
+def cli(ctx: Any, api: bool, qos: str, provider: str, specs: List[str]) -> None:
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `ctx`

(ANN401)